src/services/signatureHelp.ts

```diff
@@ -427,8 +448,11 @@ namespace ts.SignatureHelp {
         }
 
         function createSignatureHelpParameterForTypeParameter(typeParameter: TypeParameter): SignatureHelpParameter {
-            const displayParts = mapToDisplayParts(writer =>
-                typeChecker.getSymbolDisplayBuilder().buildTypeParameterDisplay(typeParameter, writer, invocation));
+            const displayParts = mapToDisplayParts(writer => {
+                const printer = createPrinter({ removeComments: true });

```

Comment:
Again, `Printer` is designed to be reused when passed the same set of initial arguments. No sense in recreating it all the time.